src/OmniSharp.Abstractions/Models/v1/ChangeBufferRequest.cs

```diff
@@ -7,9 +9,13 @@ namespace OmniSharp.Models
     public class ChangeBufferRequest : IRequest
     {
         public string FileName { get; set; }
+        [JsonConverter(typeof(ZeroBasedIndexConverter))]
```

Comment:
This line introduce two namespaces into the source code. Can we have a attribute like this to avoid it?

```
namespace Omnisharp.Json
{
    public class ZeroBasedIndexConverterAttribute: JsonConverterAttribute
    {
        public ZeroBasedIndexConverterAttribute: base(typeof(ZeroBasedIndexConverter))
        {
        }
    }    
}
```